src/lib/CMakeLists.txt

```diff
@@ -1,3 +1,8 @@
+if(BUILD_HOST)
+	add_local_sources(tb_common	lib.c )
```

Comment:
Should be space instead of tab between tb_common and lib.c, also there is space between lib.c and closing bracket